app/Http/Controllers/MovieController.php

```diff
@@ -52,4 +52,4 @@ public function destroy(Movie $movie) {
 
     }
 
-}
+}
```

Comment:
Add an empty line at the end of the file. 
The reason is that Unix historically had a convention of all human-readable text files ending in a newline, and from there a convention was born to always have an empty line.